build_tools/docker/context/setup_convperf.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Sets up `convperf` (https://github.com/nod-ai/convperf).
+#
+# `convperf` benchmarks convolution workloads on IREE and other backends such
+# as libxsmm. IREE is included as a submodule.
+#
+# Usage:
+#    ./setup_convperf.sh \
+#        <convperf repo dir> \
+#        <convperf sha>
+
+set -xeuo pipefail
+
+export REPO_DIR=$1
+export REPO_SHA=$2
+
+pushd ${REPO_DIR}
+# We use a fork of ConvPerf because the main branch relies on an outdated fork
+# of IREE and using an up-to-date version requires code changes.
+git clone --jobs 8 --depth 1 --recurse-submodules https://github.com/mariecwhite/convperf.git
+pushd convperf
+
+# Checkout a specific commit.
+git checkout ${REPO_SHA}
```

Comment:
You're likely going to hit issues here because you only cloned with depth 1 and unless the commit you want happens to be the tip of main, it's not going to be in the history here. Instead, you should do a shallow clone of a specific commit. It requires a few git commands, but is not very complicated (in fact, it's less magic than the shorthands that bundle a bunch of things at once). I mentioned this in https://github.com/iree-org/iree/pull/10956#discussion_r1029856233. Looks like those comments were not addressed

Reply:
Fixed